[Bug] CMAKE error about INTERFACE_INCLUDE_DIRECTORIES

**Bug description**
Creating the build directory, I have:
```
-- Configuring GEOSX version 0.0.31
-- Configuring done
CMake Error in coreComponents/LvArray/src/CMakeLists.txt:
  Target "lvarray" INTERFACE_INCLUDE_DIRECTORIES property contains path:

    "/home/franc90/code/geosx/GEOSX/src/../../thirdPartyLibs/install-environment-release/chai/include"

  which is prefixed in the source directory.


-- Generating done
-- Build files have been written to: /home/franc90/code/geosx/GEOSX/build-environment-release
Makefile:95: recipe for target 'rebuild_cache' failed
make: *** [rebuild_cache] Error 1
```

**To Reproduce**
Create a new build directory with the usual python script and `environment` host config.

**Expected behavior**
No errors!

**Platform (please complete the following information):**
- Ubuntu 18.04
- gcc 8.4
- cmake 3.10.2 (I tried up to 3.16.3, same outcome ...)

**Additional context**
Other users seem to not have this issue ...

--- a/src/cmake/thirdparty/setupThirdPartyDirectory.cmake
+++ b/src/cmake/thirdparty/setupThirdPartyDirectory.cmake
@@ -1,7 +1,8 @@
 
 
 if( NOT DEFINED GEOSX_TPL_ROOT_DIR )
-    set( GEOSX_TPL_ROOT_DIR "${CMAKE_SOURCE_DIR}/../../thirdPartyLibs" CACHE PATH "" )
+    get_filename_component( GEOSX_TPL_ROOT_DIR ${CMAKE_SOURCE_DIR}/../../thirdPartyLibs
+                           ABSOLUTE  CACHE )
 endif()
 
 message(STATUS "GEOSX_TPL_ROOT_DIR=${GEOSX_TPL_ROOT_DIR}")

